lib/components/map_widget.dart

```diff
@@ -87,7 +87,7 @@ class MapWidgetState extends State<MapWidget> {
   initState() {
     super.initState();
     _loadOfflineTiles();
-    _getIntentData();
+    //_getIntentData();
```

Comment:
I added a TODO here to decide between platforms or make sharing work on iOS as well